src/Library/Environment/Android/AndroidEnvironment.cpp

```diff
@@ -21,8 +21,8 @@ std::string AndroidEnvironment::path(EnvironmentPath path) const {
     return {};
 }
 
-std::string AndroidEnvironment::getenv(const std::string &key) const {
-    const char *result = SDL_getenv(key.c_str());
+std::string AndroidEnvironment::getenv(std::string_view key) const {
+    const char *result = SDL_getenv(key.data());
```

Comment:
This is not best practice, I think its okay for our use case here but will wait for one of the other devs to chime in.
Might need docs.